src/ORM/Behavior/TranslateBehavior.php

```diff
@@ -110,10 +111,12 @@ public function initialize(array $config)
             $this->_translationTable = TableRegistry::get($translationAlias);
         }
 
+        $this->config('conditions.model', $this->config('model') ?: $this->config('conditions.model') ?: $this->_table->alias());
```

Comment:
Nested ternaries are pretty convoluted to read. Some if/else would be better here.